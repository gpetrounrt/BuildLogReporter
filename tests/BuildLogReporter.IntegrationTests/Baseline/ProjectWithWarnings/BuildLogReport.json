{
  "error_count": 0,
  "warning_count": 2,
  "log_entries": [
    {
      "type": 1,
      "code": "CS8600",
      "message": "Converting null literal or possible null value to non-nullable type.",
      "file_path": "\\TestSupportProjects\\ProjectWithWarnings\\Program.cs",
      "line_number": 1
    },
    {
      "type": 1,
      "code": "CS0219",
      "message": "The variable \u0027firstAndSecondWarning\u0027 is assigned but its value is never used",
      "file_path": "\\TestSupportProjects\\ProjectWithWarnings\\Program.cs",
      "line_number": 1
    }
  ]
}