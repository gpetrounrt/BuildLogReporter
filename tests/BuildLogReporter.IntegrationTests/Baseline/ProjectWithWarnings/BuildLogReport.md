Errors: 0, Warnings: 2

| Type | Code | Message | File path | Line number |
|:---|:---|:---|:---|:---|
| Warning | CS8600 | Converting null literal or possible null value to non-nullable type. | \TestSupportProjects\ProjectWithWarnings\Program.cs | 1 |
| Warning | CS0219 | The variable 'firstAndSecondWarning' is assigned but its value is never used | \TestSupportProjects\ProjectWithWarnings\Program.cs | 1 |
