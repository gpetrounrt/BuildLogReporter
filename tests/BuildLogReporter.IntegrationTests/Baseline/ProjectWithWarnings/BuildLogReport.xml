<?xml version="1.0" encoding="utf-8"?>
<ProcessedLogResult ErrorCount="0" WarningCount="2">
  <LogEntries>
    <LogEntry Type="Warning" Code="CS8600" Message="Converting null literal or possible null value to non-nullable type." FilePath="\TestSupportProjects\ProjectWithWarnings\Program.cs" LineNumber="1" />
    <LogEntry Type="Warning" Code="CS0219" Message="The variable 'firstAndSecondWarning' is assigned but its value is never used" FilePath="\TestSupportProjects\ProjectWithWarnings\Program.cs" LineNumber="1" />
  </LogEntries>
</ProcessedLogResult>