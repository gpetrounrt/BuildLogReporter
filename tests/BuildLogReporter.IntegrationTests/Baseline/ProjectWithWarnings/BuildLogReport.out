Starting processing of '\ProjectWithWarnings\output.binlog'...
Starting extraction of log entries...
Opening binary log file...
Opened binary log file.
Processing records...
Found warning record.
Type: Warning
Code: CS8600
Message: Converting null literal or possible null value to non-nullable type.
File path: \TestSupportProjects\ProjectWithWarnings\Program.cs
Line Number: 1
Found warning record.
Type: Warning
Code: CS0219
Message: The variable 'firstAndSecondWarning' is assigned but its value is never used
File path: \TestSupportProjects\ProjectWithWarnings\Program.cs
Line Number: 1
Processed N record(s).
Completed extraction in elapsed time.
Found 0 error(s) and 2 warning(s).
Generating Badge report...
Generated report in elapsed time.
Saving report to \ProjectWithWarnings.
Saved report.
Generating Html report...
Generated report in elapsed time.
Saving report to \ProjectWithWarnings.
Saved report.
Generating Json report...
Generated report in elapsed time.
Saving report to \ProjectWithWarnings.
Saved report.
Generating Markdown report...
Generated report in elapsed time.
Saving report to \ProjectWithWarnings.
Saved report.
Generating Xml report...
Generated report in elapsed time.
Saving report to \ProjectWithWarnings.
Saved report.
Completed processing in elapsed time.
